cruise-control-client/cruisecontrolclient/client/CCParameter/SetOfChoicesParameter.py

```diff
@@ -65,5 +65,5 @@ class SubstatesParameter(AbstractSetOfChoicesParameter):
         'args': ('--substate', '--substates'),
         # nargs='+' allows for multiple of the set to be specified
         'kwargs': dict(help=description, metavar='SUBSTATE', choices=lowercase_set_of_choices, nargs='+',
-                       type=str.lower)
+                       type=str.lower, default='executor')
```

Comment:
Currently, this is set in the `__init__` method of the Endpoint definition, but I think it's more appropriate to handle the definition a layer higher which surface it to the user of the cli.